libs/packages/components/src/lib/video-player/video-player.component.ts

```diff
@@ -37,7 +37,7 @@ export class SdsVideoPlayerComponent implements AfterViewInit {
 
   ngAfterViewInit() {
     if (this.crossorigin) {
-      const id = document.getElementById('videoPlayer');
+      const id = this.elementRef.nativeElement.getElementById('videoPlayer');
```

Comment:
elementRef rather than document as document will only pick the first video if there are multiple videos in the same page